xp/mssql/Boot.ps1

```diff
@@ -0,0 +1,51 @@
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $true)]
+    [ValidateScript( {Test-Path $_ -PathType 'Container'})] 
+    [string]$InstallPath,
+    [Parameter(Mandatory = $true)]
+    [ValidateScript( {Test-Path $_ -PathType 'Container'})] 
+    [string]$DataPath
+)
+
+$noDatabases = (Get-ChildItem -Path $DataPath -Filter "*.mdf") -eq $null
+
+if ($noDatabases)
+{
+    Write-Host "### Sitecore databases not found in '$DataPath', seeding clean databases..."
+
+    Get-ChildItem -Path $InstallPath | ForEach-Object {
+        Copy-Item -Path $_.FullName -Destination $DataPath
+    }
+}
+else
+{
+    Write-Host "### Existing Sitecore databases found in '$DataPath'..."
+}
+
+Get-ChildItem -Path $DataPath -Filter "*.mdf" | ForEach-Object {
```

Comment:
Waarom is dit nodig?

Reply:
Dit attached alle databases uit de directory (-Path)